src/main/java/forum/entity/Category.java

```diff
@@ -19,6 +25,12 @@
 	@Column(name="title")
 	private String title;
 	
+	/*
+	@OneToMany(fetch=FetchType.EAGER, mappedBy="category",
```

Comment:
Commented code should be removed

Reply:
Yeah I forgot to remove it. All gone now!